src/ILogger/ApplicationInsightsLoggerFactoryExtensions.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace Microsoft.Extensions.Logging
+{
+    using System;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.ILogger;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Options;
+
+    /// <summary>
+    /// Extension methods for <see cref="ILoggerFactory"/> that allow adding Application Insights logger.
+    /// </summary>
+    public static class ApplicationInsightsLoggerFactoryExtensions
```

Comment:
I'd like to remove these extension methods completely and use the ILoggerBuilder pattern. It better fits what AI because the telemetry client needs to be added.